src/styles/variables.json

```diff
@@ -60,74 +60,12 @@
     "bdl-watermelon-red-10": "#fdebee",
     "bdl-watermelon-red-05": "#fef5f6",
     "bdl-watermelon-red-02": "#fffbfc",
-    "bdl-secondary-blue": "#003c84",
-    "bdl-neutral-01": "#222",
-    "bdl-neutral-02": "#767676",
-    "bdl-neutral-03": "#909090",
-    "bdl-neutral-04": "#e8e8e8",
-    "bdl-neutral-05": "#f4f4f4",
-    "bdl-neutral-06": "#fbfbfb",
-    "box-blue": "#0061d5",
```

Comment:
We're sure that these values aren't being used anywhere? Are we expecting consumers to import `deprecated.*` files directly?

Reply:
- `variables.json` is only used in `styleguide.config.js`
- `variables.js` is used by js code but Flow can tell us what is used. This PR fixes anything in BUIE affected by the change, and EUA already has removed the deprecated values.
- `_variables.scss` continues to re-export the deprecated values for the few places that haven't been ported over.